index.html

```diff
@@ -14,7 +14,7 @@
         <!-- ============ Materialize core CSS ============ -->
```

Comment:
This also wasn't you, but these comments are superfluous and not very useful. We can see that it is importing materialize's css, we don't need 2 lines telling us the same thing.